tests/cypress/integration/case_5_image_rotate.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check if the image is rotated', () => {
+
+    const caseId = '5'
+    const labelName = `Case ${caseId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`
+    const image = `${imageFileName}.png`
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    function rotateClockwise() {
+        cy.get('.cvat-canvas-container')
+        .trigger('keydown', {key: 'Control'})
+        .trigger('keydown', {key: 'r'})
+        .trigger('keyup', {key: 'r'})
+        .trigger('keyup', {key: 'Control'})
```

Comment:
Let's use UI buttons for that in this test

Reply:
Could you please add class to this buttons? Also to ROI button (it will be useful for the next test)

Reply:
Sure

Reply:
Done #2197. I will merge it after CI is passed

Reply:
Thanks. Will rebase and apply comments.

Reply:
Replaced image rotation via buttons.

Reply:
Great, thx